pkg/cable/vxlan/vxlan.go

```diff
@@ -97,6 +97,10 @@ func NewDriver(localEndpoint *types.SubmarinerEndpoint, localCluster *types.Subm
 		localCluster:  *localCluster,
 	}
 
+	if strings.ToLower(localEndpoint.Spec.CableName) == CableDriverName && localEndpoint.Spec.NATEnabled {
```

Comment:
```
pkg/cable/vxlan/vxlan.go:100:5: equalFold: consider replacing with strings.EqualFold(localEndpoint.Spec.CableName, CableDriverName) (gocritic)
	if strings.ToLower(localEndpoint.Spec.CableName) == CableDriverName && localEndpoint.Spec.NATEnabled {
	   ^
```

This is flagged by Go linting

https://github.com/submariner-io/submariner/actions/runs/4485520060/jobs/7926833046?pr=2346

Reply:
Thanks for the reply, Updated!  No push, I'm feel ok :)